Playbooks/Get-GeoFromIpAndTagIncident/customConnector/azuredeploy.json

```diff
@@ -6,29 +6,43 @@
+                "description": "Enter the IP-API Endpoint Url"
+            }
+        }
     },
     "variables": {
     },
     "resources": [
         {
             "type": "Microsoft.Web/customApis",
             "apiVersion": "2016-06-01",
-            "name": "IP-API",
+            "name": "[parameters('ConnectorName')]",
             "location": "[resourceGroup().location]",
             "properties": {
                 "backendService": {
-                    "serviceUrl": "https://ip-api.com/json"
+                    "serviceUrl": "[parameters('ServiceEndPoint')]"
                 },
-                "displayName": "IP-API",
+                "displayName": "[parameters('ConnectorName')]",
                 "iconUri": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAhCAYAAAC4JqlRAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAB3RJTUUH5QYDFTM5piv2fAAABGZJREFUWIWl12mMnVMYB/Df3BlubUeiWlVLKaWKBpVWY2lRPqAhNLG1IbHTCGL7IPhAUrFEhNiXCNogqNiaIEJjCRV7aGjFvpTyVOmgHR/OufrONZ17Z/yTyTvvc573/J97nvV0aEJKaRwuwjQMxU94HtdFxIdFZ31shk3xN5ZhRUSsad6vFTqayE/Gbaj3oduNs/EeXkCqrL2Pb7EKz+HRiPixHQNqFfKpuLNCvgLvlqcivwObYF6R/YyzMBGvYTi+wfSU0tB2DOgo5PAm9i7yu3F+RKxIKW2MG3BaWVuEE8rzDzyNLfA7ZkXEH+0QNxuwHZaU9w+wR0SsbiillDrxNsajBzvKLvlVjoHuiOhJKS3AGpwWEV+1Y0DDBaOtjYcFVXIo7wsqRo+OiK8j4reIWIWJKaX9cA52xcKU0kYDMWBVRbb5OnSHVf7/vWntZDyL5ThEzo6j2jGg4YKh+B6dWIkJEfFJQymlNEZ2wcb4CyMi4ueSjtfgWFyOsRiJY+QAPjQifil7dKALwyLim14GFIXXsE95/QFz5JTbHZfKgQYvRcSBlU3nYXhDVuR3yEH7Lg4vf1fIdWV9TI2Il6snsAFmymnWCidGxEP9KaSURmIhtsePGCKnbwNvYHJE9NRKCt6DJ/FFC/KP8XArC8sRH1T2G9ZEDpMwBTrr9foo3I4/8RBmrGPfHpwUEYtbGZBS2go7IXDAuvS6u7vnd+EwOfguxF6YjyP70L9fjvT+iEfIfeRMbNjCzsNSSp1d2K8IhuBeOYL3xLYV5cU4NyL6I98Gr8tZ0A6GY0wN4yrCSfJJzJDTkXyMM6If9lKuHx8AeQNjaxjRJDxPPoHj8Juc47s2V7aU0gYppX1Lmb4YEwZIDlvW9N16b5ZdMlZuuw/gxpTS8JI18BiewCdyCR4MhnTJ0d+M9fAgbpXbbSdOkf12kzwPXCK7Z+kgyWFVZ71eP9F/3aCQ7lOe5KK1MybX6/Wh+AWf4lVMx5dyZ9xQ06DTD+7uSCnNlf09GCzCoYV4RSHeTi5CU+Tq2h/G1eRfMFhMwEfYKyJ6ImJNRCyJiLswC7f08+0yLK7hGXmIGCxq8jTVCyVrz5MzpK8UfjYiVtfwGV75HwbMi4iVfS1ExN8RcS12kXtNFfdBrVh6/f8w4JhW009pTtUp6y28xNqJ6KmGYBAYidn9KaSUJurdXy5p3CFqxcIenCFXvsHg+EqBaibvwFXW/ti7IuLFxvq/94LSZk83uIDcDdv0Qb6v7PtpRbRIDsx/0dX0zVx59LpB+8WEXKy2lotRg3xb2a0Njo9xRHPA1qovJSBvxKl6T8qtsBLvNMlmVsgXYkpEfNf8Ya1ZEBEi4h7sL19S2sEbjRtRSmlISmkOrpQn6KtxcET80NeH/zGgYshb8oQ0G5+3MGBpIR8j+/kCPIrxEXFZRPTV8NCmn1NK68mBdJRc43fQO36ejIgjU0pHYxQeafdqNpBAaxhD7ngjrL2ofBcRywe6F/wDhBx6oUpxUmgAAAAASUVORK5CYII=",
                 "swagger": {
                     "swagger": "2.0",
                     "info": {
-                        "title": "IP-API",
+                        "title": "[parameters('ConnectorName')]",
                         "description": "IP-API Geolocation API",
                         "version": "1.0"
                     },
-                    "host": "ip-api.com",
+                    "host": "[replace(replace(parameters('ServiceEndpoint'),'https://',''),'http://','')]",
                     "basePath": "/json",
                     "schemes": [
                         "https"
```

Comment:
Schemes should also be http instead of https.

Reply:
done